intellij/src/saros/intellij/ui/views/buttons/ConsistencyButton.java

```diff
@@ -108,15 +80,33 @@ public ConsistencyButton(@NotNull Project project) {
         Messages.ConsistencyButton_tooltip_functionality,
         IconManager.IN_SYNC_ICON);
 
-    SarosPluginContext.initComponent(this);
-
     this.project = project;
 
-    setSarosSession(sessionManager.getSession());
-    sessionManager.addSessionLifecycleListener(sessionLifecycleListener);
+    setSarosSession(sarosSessionManager.getSession());
 
     addActionListener(actionListener);
     setEnabled(false);
+
+    setInitialState();
```

Comment:
I normally would expect by that this method name that it also sets the inconsistency state, however it does not. Add a TODO ?

Reply:
Thanks for the note. I just forgot. Fixed.